calculator.py

```diff
@@ -65,6 +65,28 @@ def exp():
         return result
     else:
         print("入力値が正しくありません")
+
+def calculate_twice():
```

Comment:
```suggestion
def calculate_twice():
こんな感じのコードだととりあえずうごきます。
まだ例外処理とか入れていないので考えてみます。
```
def calculate_twice():
    """
    ユーザーに処理したい演算と値を入力してもらい、直前の演算結果に対する演算結果を返す（演算は、加算・減算・乗算・除算の中から選択）
    """
    sub_choice = input("演算項目を選択してください(1-4): ")
    if sub_choice in ["1", "2", "3", "4"]:
        num = input("演算を行う数値を入力してください: ")
        pre_result = result
        if is_numeric(num):
            if sub_choice == "1":
                result = pre_result + num
            elif sub_choice == "2":
                result = pre_result - num
            elif sub_choice == "3":
                result = pre_result * num
            elif sub_choice == "4":
                result = pre_result / num
            print(f"計算結果 : 直前の結果 {result} に対する演算 {sub_choice} を行った結果 = {result}")
            return result

    else:
        print("入力値が正しくありません")

Reply:
こんな感じだと動きました。

Reply:
@momokowv 
直前の計算結果が計算後の計算結果が表示されるようになっていたので少し変えました。
```suggestion
def calculate_twice(result):
    sub_choice = input("演算項目を選択してください(1-4): ")
    pre_result = result
    if sub_choice in ["1", "2", "3", "4"]:
        num = input("演算を行う数値を入力してください: ")
        if is_numeric(num):
            if sub_choice == "1":
                result += float(num)
            elif sub_choice == "2":
                result -= float(num)
            elif sub_choice == "3":
                result *= float(num)
            elif sub_choice == "4":
                result /= float(num)
            print(f"計算結果 : 直前の結果 {pre_result} に対する演算 {sub_choice} を行った結果 = {result}")
            return result
        else:
            print("入力値が正しくありません")
```